indexer/restriction_loader.cpp

```diff
@@ -0,0 +1,51 @@
+#include "indexer/restriction_loader.hpp"
+
+#include "coding/reader.hpp"
+
+using namespace routing;
+
+namespace
+{
+template <class TSource>
+void DeserializeRestrictions(Restriction::Type type, uint32_t count, TSource & src,
```

Comment:
Yes. Have a look at the commit "Reorganization restriction serialization and deserialization.". It's done in it.